pkg/sshutil/sshutil.go

```diff
@@ -332,6 +344,10 @@ func ParseOpenSSHVersion(version []byte) *semver.Version {
 	return &semver.Version{}
 }
 
+func ParseOpenSSHGSSAPISupported(version string) bool {
```

Comment:
Since this function is used only inside the package, we can make it unexported:
```suggestion
func parseOpenSSHGSSAPISupported(version string) bool {
```

Reply:
Thanks! Updated in https://github.com/lima-vm/lima/compare/4daab365eff3fec89c9f4ff5e7838c6f35cbbc0d..1fbf86db9443418409f31576ed2b1c425a0f736c